heartbeat/monitors/active/dialchain/builder.go

```diff
@@ -137,23 +137,30 @@ func MakeDialerJobs(
 ) ([]monitors.Job, error) {
 	var jobs []monitors.Job
 	for _, endpoint := range endpoints {
-		endpointJobs, err := makeEndpointJobs(b, typ, scheme, endpoint, mode, fn)
+		/*
+			endpointJobs, err := makeEndpointJobs(b, typ, scheme, endpoint, mode, fn)
+			if err != nil {
+				return nil, err
+			}
+			jobs = append(jobs, endpointJobs...)
+		*/
```

Comment:
let's remove code, instead of commenting it out.

Reply:
Ah, my mistake, sorry!